app/lib/konnectors.js

```diff
@@ -49,8 +49,8 @@ function waitForKonnectorReady (cozy, konnector, timeout) {
 
     const idInterval = setInterval(() => {
       cozy.data.find(KONNECTORS_DOCTYPE, konnector._id)
-        .then(konnector => {
-          if (konnector.state === STATE_READY) {
+        .then(konnectorResult => {
```

Comment:
I think we should keep `konnector` here (and resolve the promise with it). Is it causing any issue ?